src/aklite_client_ext.cc

```diff
@@ -129,8 +129,10 @@ GetTargetToInstallResult AkliteClientExt::GetTargetToInstall(const CheckInResult
           << res.selected_target.Name();
 
       res.status = GetTargetToInstallResult::Status::UpdateSyncApps;
-      res.reason =
-          "Syncing Active Target Apps\nApps requiring update:\n" + boost::algorithm::join(apps_to_update, "\n");
+      res.reason = "Syncing Active Target Apps\n";
```

Comment:
I think we can combine the last two commits into a single commit as they change the same code lines twice.

Reply:
Done.